hack/linter.sh

```diff
@@ -24,7 +24,7 @@ golangci-lint run \
     -E gosec \
     -E gofmt \
     --skip-dirs vendor \
-    --deadline 5m0s \
+    --deadline 15m0s \
```

Comment:
15min !? Does linting take so long?

Reply:
It's about 5 min right now. And it seems to timeout occasionally resulting in a build failure.